### PR TITLE
Update setup to reflect latest stable release

### PR DESCRIPTION
**Pull request by MarkBennett**
Couldn't figure out why I was getting old code until I looked at my Gemfile!

### CODE REVIEW

**Comment by smathieu** [comment]
We probably shouldn't show a version in the Gemfile. Can you simply remove it and I'll merge? Thanks for the PR.

File: README.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ $ ruby -r queue_classic -e "QC::Worker.new.work"
 Declare dependencies in Gemfile.
 ```ruby
 source "https://rubygems.org"
-gem "queue_classic", "~> 3.0.0"
+gem "queue_classic"
```
Comment (MarkBennett) [inline README.md]: Is this what you mean @smathieu?

Comment (jipiboily) [inline README.md]: Yup, exactly. 😀

**Comment by jipiboily** [comment]
Thanks @MarkBennett!

**Pull request merged**